### PR TITLE
make operator-sdk local bin

### PR DESCRIPTION
**Pull request by yazug**
specify version for operator-sdk for make bundle
download operator-sdk pinned version when generating bundle update make bundle to use local operator-sdk

### CODE REVIEW

**Review by fmount — APPROVED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -237,27 +241,48 @@ CONTROLLER_TOOLS_VERSION ?= v0.11.1
 
 KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
 .PHONY: kustomize
-kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
+kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary. If wrong version is installed, it will be removed before downloading.
 $(KUSTOMIZE): $(LOCALBIN)
```
Comment (ASBishop) [inline Makefile:245]: As I noted in https://github.com/openstack-k8s-operators/cinder-operator/pull/308, the KUSTOMIZE target needs to be phony

**Review by ASBishop — APPROVED** [review]

**Comment by yazug** [comment]
/retest 
Failed trying to fetch install_yamls repo

File: Makefile (hunk 1 above)
Comment (yazug) [reply Makefile:246]: addressed

**Review by fmount — APPROVED** [review]
/lgtm

**Pull request merged**